internal/controller/standalone_pgadmin/pod.go

```diff
@@ -50,11 +52,21 @@ func pod(
 		Sources: podConfigFiles(inConfigMap, *inPGAdmin),
 	}
 
+	pgAdminLog := corev1.Volume{Name: logVolume}
+	pgAdminLog.EmptyDir = &corev1.EmptyDirVolumeSource{
+		Medium: corev1.StorageMediumMemory,
+	}
+
+	tmpVol := corev1.Volume{Name: "tmp"}
+	tmpVol.EmptyDir = &corev1.EmptyDirVolumeSource{
+		Medium: corev1.StorageMediumMemory,
+	}
```

Comment:
Needed to add a vol for logging, or maybe direct logging to the data dir?
(Tmp vol seemed like a useful thing to have.)

Reply:
:thinking: I like the idea of a separate logging volume so we don't clobber the data directory. I partially think a separate, non-tmp volume would be nice. Maybe something to discuss? As an intermediate option, perhaps we could leave it as a tmp volume for now and put a `TODO` about investigating a more permanent volume option?

Reply:
Added a TODO